modules/update-upgrading-cli.adoc

```diff
@@ -181,10 +181,10 @@ $ oc get clusterversion
 ----
 +
 .Example output
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
 NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
-version   4.10.13      True        False         2m        Cluster version is 4.10.13
+version   {product-version}      True        False         2m        Cluster version is {product-version}
```

Comment:
Discussed w/ @sheriff-rh about losing the z-stream version number in the output here, and agree with him that this route is the least of the evils (unless we created / started bumping a new common-attribute each z-stream release).